src/main.ts

```diff
@@ -319,239 +184,63 @@ async function moveToQuadrant(motionClient: MotionClient, armClient: ArmClient,
-  console.log('current position:' + JSON.stringify(currentPosition))
-  let rightPose: Pose = {
-    x: currentPosition.pose!.x,
-    y: currentPosition.pose!.y + moveDistance,
-    z: currentPosition.pose!.z,
-    theta: 0,
-    oX: 0,
-    oY: 0,
-    oZ: -1
-  };
-
-  let rightPoseInFrame: SDK.PoseInFrame ={
-    referenceFrame: "world", 
-    pose: rightPose
-  }
-
-  await motionClient.move(rightPoseInFrame, myResourceName, myWorldState, constraints)
-}
-
-async function left(motionClient: MotionClient, armClient: ArmClient) {
-  if (ignoreInterrupts && await armClient.isMoving()) { console.log("Too fast!"); return }
-  
-  //Create a WorldState that has Geometries in Frame included 
-  let myObstaclesInFrame: SDK.GeometriesInFrame = {
-    referenceFrame: "world", 
-    geometriesList: [leftWallObject],
-  }
-  
-  let myWorldState: SDK.WorldState ={
-    obstaclesList: [myObstaclesInFrame],
-    transformsList: [],
-  }
-
-  //Get current position of the arm 
-  console.log('im trying to print the current position')
-  let currentPosition = await motionClient.getPose(myResourceName, 'world', [])
-  console.log('current position:' + JSON.stringify(currentPosition))
-  let leftPose: Pose = {
-    x: currentPosition.pose!.x,
-    y: currentPosition.pose!.y - moveDistance,
-    z: currentPosition.pose!.z,
-    theta: 0,
-    oX: 0,
-    oY: 0,
-    oZ: -1
-  };
-
-  let leftPoseInFrame: SDK.PoseInFrame ={
-    referenceFrame: "world", 
-    pose: leftPose
-  }
-
-  await motionClient.move(leftPoseInFrame, myResourceName, myWorldState, constraints)
-}
-
-async function dropDown(motionClient: MotionClient, armClient: ArmClient) {
-  if (ignoreInterrupts && await armClient.isMoving()) { return }
-
-  //Create a WorldState that has Geometries in Frame included 
-  let myObstaclesInFrame: SDK.GeometriesInFrame = {
-    referenceFrame: "world", 
-    geometriesList: [holeObject],
-  }
-  
-  let myWorldState: SDK.WorldState ={
-    obstaclesList: [myObstaclesInFrame],
-    transformsList: [],
-  }
-
-  //Get current position of the arm 
-  console.log('im trying to print the current position')
-  let currentPosition = await motionClient.getPose(myResourceName, 'world', [])
-  console.log('current position:' + JSON.stringify(currentPosition))
-
-  let dropPose: Pose = {
-    x: currentPosition.pose!.x,
-    y: currentPosition.pose!.y,
-    z: 240,
-    theta: 0,
-    oX: 0,
-    oY: 0,
+    oY: 0, 
     oZ: -1
   };
-
-  let dropPoseInFrame: SDK.PoseInFrame ={
+  let pif: SDK.PoseInFrame = {
     referenceFrame: "world", 
-    pose: dropPose
+    pose: pose
   }
 
-  //Drop the claw down
-  console.log('im about to drop to' + JSON.stringify(dropPoseInFrame))
-  await motionClient.move(dropPoseInFrame, myResourceName, myWorldState, constraints)
-  console.log('dropped')
-
+  // Move to new position
+  console.log('moving to:' + JSON.stringify(pif))
+  await motionClient.move(pif, armName, myWorldState, constraints)
```

Comment:
@nfranczak there was talk of manually setting the most distal joint to 0 after this move call, that should go here, no?